cpp/server/ct-server.cc

```diff
@@ -86,6 +86,7 @@ DEFINE_double(guard_window_seconds, 60,
               "number of seconds will not be sequenced.");
 DEFINE_string(etcd_host, "", "Hostname of the etcd server");
 DEFINE_int32(etcd_port, 0, "Port of the etcd server.");
+DEFINE_string(etcd_root, "/root", "Root of cluster entries in etcd.");
```

Comment:
I'll save that for another PR because it'll also involve changing some of the GCE scripts etc.